src/App.js

```diff
@@ -1,10 +1,19 @@
-import React, { useState } from "react";
+import React, { useState , useEffect} from "react";
 import logo from "./logo.svg";
```

Comment:
I think you could remove this import since you don't use it, to keep your code clean 🧽

```suggestion
import logo from "./logo.svg";
```